.devcontainer/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.166.1/containers/go/.devcontainer/base.Dockerfile
```

Comment:
Are the `.devcontainer` files meant to be here or something from your local dev environment?

Reply:
oops, removed